noir-projects/aztec-nr/aztec/src/context/call_interfaces.nr

```diff
@@ -51,68 +51,10 @@ where
         }
     }
 
```

Comment:
The docs from this file were moved over to `ContractSelf` and I marked the functions as deprecated to point devs to the new API.